src/components/MediaPlayer/MainControl/VolumeBar/VolumeBarComponent.js

```diff
@@ -0,0 +1,103 @@
+import volumeBarTemplate from "./VolumeBarComponent.html";
+
+export class VolumeBarComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+    this.volumeDefault = 0.6;
+  }
+
+  querySelectors() {
+    this.audio = document.querySelector(".player-buttons__main-audio");
```

Comment:
I'm not sure, but can you check about "this.mountPoint.querySelector" instead of "document.querySelector"

Reply:
++